pages/code.md

```diff
@@ -632,90 +659,204 @@ class MLStudioApiModuleInterface(ModuleInterface):
+The frontend part of this project is developed using React.js and bootstrapped using [Create React App](https://github.com/facebook/create-react-app). You can run the app in development mode using `yarn start`, and open [http://localhost:8080](http://localhost:8080) to view it in the browser.
+
+Overview of the frontend directory:
+```
+frontend/
+├── docker/
+├──── dev/
+├────── Dockerfile : contains the docker config for the dev environment
+├────── nginx.config : contains the nginx config for the dev environment
+├──── prod/
+├────── Dockerfile : contains the docker config for the prod environment 
+├────── nginx.config : contains the nginx config for the prod environment
+├── public/ : contains static files such as index.html and icons like the app logo and favicon
+├── src/ : contains the major components of the frontend code, to be discussed in detail in the src/ section below
+└── jsconfig.json : used to reference the root folder and map aliases/modules to their respective paths relative to the root folder.
+```
+
+### src/ <a name="src"></a>
+This section contains the major components of the frontend code. Here is a short description of all the components of the src folder, we will deep dive subsequently on the contents of each module.
+
+```
+src/
+├── authentication/ : contains files, contexts, hooks related to user and guest authentication
+├── design/ : contains scripts related to the ui design of the app, layout and theme settings
+├── globalErrors/ : global error reducers, uses redux
+├── modules/ : contains directories of each view/screen in the app with their related components, hooks and services
+├── services/ : contains common graphql schemas and hooks used to call the backend APIs which are then plugged into the frontend
+├── utils/ : common helpers used across the app
+├── App.js - entry point into the project, wrapped with the theme provider
+├── index.js - react.js index script, wrapped with several providers
+└── routes.js - where all routes and their hierachies are configured
+```
+#### authentication/ <a name="authentication"></a>
+In this section, we handle the user and guest authentication logic and views for the application. The directory contains React `contexts`, `hooks`, `components` and `views` used to handle local and production environment authentication. 
+
+```
+autentication/
+├── components/
+├── contexts/
+├── hooks/
+├── views/
+└── index.js
+```
+
+We used React Context API to handle the authentication state mangagement. `CognitoAuthContext.js` handles the AWS deployment and `LocalAuthContext.js` for local deployment.
+Then `AWS Amplify` is used to connect the app to `AWS Cognito` for production auth and a default anonymous user  is set for local environment.
+
+The `useAuth` hook is used to decide the auth context to use depending on the deployment environment, and `AuthGuard` is a wrapper to verify authentication before routing users to the requested pages in the application.
+
```

Comment:
I think lines 711 to 714 should belong to other parts of the documentation (it is more like user guide, getting started...) here I would focus on code only

Reply:
Okay this will be refactored out

Reply:
updates added here https://github.com/awslabs/aws-dataall/pull/722/commits/76b8b7b3178a65553b88b72164562121b76bfde5